### ISSUE TITLE
Minimal nodejs version?

### ISSUE THREAD

**Issue by beepsoft** [mweststrate/relative-deps#23]
Hi,

it seems that relative-deps doesn't work with nodejs 8 only 10 anc above.

With nodejs 8 I get such errors:

```
> yarn
yarn install v1.22.0
[1/4] 🔍  Resolving packages...
success Already up-to-date.
$ relative-deps
...node_modules/relative-deps/index.js:259
      } catch {
              ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (.../node_modules/relative-deps/cli.js:3:22)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

I coudn't really figure out what could be the actual problem causing this exception.

Do you plan to support nodejs 8?

**Comment by elektronik2k5** [comment]
Considering the fact that node@8 is deprecated, I vote for not supporting it.
Generally speaking, nobody should be using deprecated and vulnerable software.

**Comment by beepsoft** [comment]
@elektronik2k5 I agree with you, but I think the minimum supported node version should then be mentioned in the README.

**Comment by mweststrate** [comment]
In principle 8 is not supported, but pragmatically replacing it with
"catch(_e)" might be enough to make node 8 work, if it does, feel free to
PR.

Op zo 9 feb. 2020 17:56 schreef beep <notifications@github.com>:

**Comment by mweststrate** [comment]
Solved in 1.0.2
